### PR TITLE
Add Stockfish 17 Release

### PR DESCRIPTION
**Pull request by Disservin**

### CODE REVIEW

**Comment by Disservin** [comment]
Anything else which needs to be changed? I think all file name's are the same, or do we have more binaries which will be released for sf 17 than we had for sf 16.1?

**Comment by dav1312** [comment]
I don't think so. I already made a lot of changes in the previous release to make new ones as straightforward as possible.

**Comment by Disservin** [comment]
Yes I noticed that :D Well done ;)

File: content/blog/2024/stockfish-17.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+## Update highlights
+
+### Improved engine lines
+
+This release introduces principal variations (PVs) that are more informative for mate and decisive table base (TB) scores. In both cases, the PV will contain all moves up to checkmate. For mate scores, the PV shown is the best variation known to the engine at that point, while for table base wins, it follows, based on the TB, a sequence of moves that preserves the game outcome to checkmate.
+
+### NUMA performance optimization
+
+For high-end computers with multiple CPUs (typically a dual-socket architecture with 100+ cores), this release automatically improves performance with a \`NumaPolicy\` setting that optimizes non-uniform memory access (NUMA). Although typical consumer hardware will not benefit, [speedups of up to 2.8x][7] have been measured.
```
Comment (vondele) [inline content/blog/2024/stockfish-17.md:24]: escaping

File: content/blog/2024/stockfish-17.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+## Shoutouts
+
+### ChessDB
+
+During the past 1.5 years, hundreds of cores have been continuously running Stockfish to grow a database of analyzed positions. This [chess cloud database][8] now contains well over 45 billion positions, providing excellent coverage of all openings and commonly played lines. This database is already integrated into GUIs such as [En Croissant][9] and [Nibbler][10], which access it through the public API.
+
+### Leela Chess Zero
+
+Generally considered to be the strongest GPU engine, it continues to provide open data which is essential for training our NNUE networks. They [released version 0.31.1][11] of their engine a few weeks ago, check it out\!
```
Comment (vondele) [inline content/blog/2024/stockfish-17.md]: escaping

File: content/blog/2024/stockfish-17.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+### Website redesign
+
+Our website has undergone a redesign in recent months, most notably in our [home page][12], now featuring a darker color scheme and a more modern aesthetic, while still maintaining its core identity. We hope you'll like it as much as we do\!
```
Comment (vondele) [inline content/blog/2024/stockfish-17.md]: escaping

File: content/blog/2024/stockfish-17.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+## Thank you
+
+The Stockfish project builds on a thriving community of enthusiasts (thanks everybody\!) who contribute their expertise, time, and resources to build a free and open-source chess engine that is robust, widely available, and very strong.
+
+We would like to express our gratitude for the [11k stars][13] that light up our GitHub project\! Thank you for your support and encouragement – your recognition means a lot to us.
```
Comment (vondele) [inline content/blog/2024/stockfish-17.md]: escaping

File: content/blog/2024/stockfish-17.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+Generally considered to be the strongest GPU engine, it continues to provide open data which is essential for training our NNUE networks. They [released version 0.31.1][11] of their engine a few weeks ago, check it out\!
+
+### Website redesign
+
+Our website has undergone a redesign in recent months, most notably in our [home page][12], now featuring a darker color scheme and a more modern aesthetic, while still maintaining its core identity. We hope you'll like it as much as we do\!
+
+## Thank you
+
+The Stockfish project builds on a thriving community of enthusiasts (thanks everybody\!) who contribute their expertise, time, and resources to build a free and open-source chess engine that is robust, widely available, and very strong.
```
Comment (vondele) [inline content/blog/2024/stockfish-17.md]: escaping

**Pull request merged**